.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimexUtility.cs

```diff
@@ -642,6 +642,96 @@ public static int ParseHourFromTimeTimex(string timex)
             return new Tuple<int, int>(hour1, hour2);
         }
 
+        public static TimeOfDayResolutionResult ResolveTimeOfDayTaskMode(string tod)
```

Comment:
Wouldn't it be better in the task-specific utils?

Reply:
I have moved mentioned function inside TasksModeProcessing.cs file.